refs/schemas/priv/RFC-PRIV.md

```diff
@@ -476,6 +479,59 @@ It may happen that two or more Retention Policies conflict, e.g. the same data m
 
 In such cases, `NO-LESS-THAN` takes priority over `NO-LONGER-THAN`. The data is kept.
 
+## Lifecycle of Legal Bases for data processing
+
+Each System SHOULD have a knowledge (i.e. a configuration) of a set of [Legal Base](#legal-base) objects.
+Those objects define, for that particular System, what makes it legal to process particular data categories in a particular way for a particular purpose.
+For example, a System might be configured to store and use (`processing-category`:{`STORING`, `USING`}) `CONTACT` data under the `CONTRACT` Legal Base because it needs the data in order to perform a service for the Data Subject.
+The same System might be configured to share (`processing-category`:`SHARING`) `CONTACT` data only under `CONSENT` Legal Base, i.e. if the Data Subject explicitly approves it.
+
+Once the System knows the Legal Bases for each [Privacy Scope](#privacy-scope), it relies on [Legal Base Events](#legal-base-event) and [Consents](#consents) to determine, for each user, the active Legal Bases. E.g. when a Data Subject starts using the System services (`SERVICE-START` type of event) the Privacy Scope associated with `CONTRACT` Legal Base becomes active.
+
+For more details, refer to [Expected Behavior of Implementing Systems](./expected-behavior.md).
+
+### Legal Base
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `legal-base` | 1-* | One or more [Legal Base Term(s)](#legal-bases)|
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) covered by the given Legal Base(s). In absence of any indication the Event is considered to concern all the possible scope. |
+
+
+### Legal Base Event
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-subject` |  1-* | Data Subject concerned by the event, identified by one or more of their [Data Subject Identities](#decentralized-identity-of-data-subjects) each containing one `dsid` and one `dsid-schema`|
+| `event-type` | 1 | Any of the [Event Terms](#events) |
+| `legal-base` | 1-* | Any [Legal Base Term](#legal-bases) other than `CONSENT`|
+| `data-reference` | 0-* | optionally one or more references that uniquely identify the data that the event concerns (e.g. a legal case file reference, account ID, contract ID, a URL - For example, if the user canceled a particular subscription contract it is the contract ID) |
+| `date` | 1 | Date and Time of the Event given in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) gained or lost by this particular event. Specified only for special cases where a particular event affects only a part of the scope of the Legal Base. If not specified, the total Privacy Scope of the Legal bases is assumed. |
```

Comment:
The idea here, in most cases `scope` will not be sepcified and will mean the whole scope that is defined in the [configuration](https://github.com/blindnet-io/product-management/blob/8fa3b1bb35b0404f21da5a31f2ced5118e5b9f01/refs/schemas/priv/expected-behavior.md#configuration-and-prerequisites) as being covered by a particular legal base.

However, you may have particular contracts that make it legal for you to hold a particular fragment, and other contract for another fragment. In scuh case, you can optionally specify the sub-`scope` here. 

For the privacy compiler it should be the same - it just takes either the scope of the Legal Base from [configuration](https://github.com/blindnet-io/product-management/blob/8fa3b1bb35b0404f21da5a31f2ced5118e5b9f01/refs/schemas/priv/expected-behavior.md#configuration-and-prerequisites) or this scope (if defined). The only complexity I see is that we must check that indeed this scope is subset of the scope from the configuration, and it can create issues. 

You might be right that the tread-off is (at the moment) in favour of not allowing to define a sepcific `scope`.

@m4rk055  Do you still think the same after my comment?

Reply:
```suggestion
```
@m4rk055 commit if agree